app/assets/javascripts/Components/Modals/collect_submissions_modal.jsx

```diff
@@ -59,6 +73,35 @@ class CollectSubmissionsModal extends React.Component {
                 />
               </label>
             </p>
+            <p>
+              <label>
+                <input
+                  type="checkbox"
+                  name="collect_current"
+                  onChange={this.handleCollectCurrentChange}
+                />
+                &nbsp;
+                <span
+                  dangerouslySetInnerHTML={{
+                    __html: I18n.t("submissions.collect.collect_current"),
+                  }}
+                />
+              </label>
+            </p>
+            <p>
+              <label>
+                <input
+                  type="checkbox"
+                  defaultChecked={this.state.apply_late_penalty}
+                  name="apply_late_penalty"
+                  id="apply_late_penalty"
```

Comment:
Is there a reason why we need an id here but not for the other checkboxes? Was there some custom css you wanted to apply to this one?

Reply:
I was at one point intending to use css to hide this checkbox unless the `collect_now` checkbox was checked, but I realized that didn't really make sense so I removed it and this got left in accidentally. I've removed it since it isn't actually doing anything.